packages/front-end/components/SavedGroups/SavedGroupForm.tsx

```diff
@@ -57,9 +60,15 @@ const SavedGroupForm: FC<{
       type,
       values: current.values || [],
       description: current.description || "",
+      projects: current.projects || project ? [project] : [],
```

Comment:
Something about this isn't working and I haven't identified why yet. I added a project to a saved group then went to edit it and the current group isn't already in the multiselect. Hopefully these debug logs are helpful:

```
  console.log(
    "Rendering savedGroupForm. Current projects are",
    current.projects
  );
  console.log("Form projects are", form.watch("projects"));

```

![image](https://github.com/user-attachments/assets/5690459a-4292-4ad1-b15a-afdfcc5edc30)

Reply:
Oh yeah, found it: you just need parenthesis on the logic like this: `current.projects || (project ? [project] : [])`